lib/services/fetch_product_from_storage.rb

```diff
@@ -0,0 +1,8 @@
+module Store
+  class FetchProductFromStorage
+    def call(id, storage)
+      return unless product = storage.find { |item| item.product_id == id.to_i }
```

Comment:
Why `nil` is accepted while `FetchProductById#call` will raise an exception if item is missing? How is it different from that service?